src/Rules/FutureDate.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Spatie\ValidationRules\Rules;
+
+use Spatie\ValidationRules\IsDateRule;
+use Illuminate\Contracts\Validation\Rule;
+
+class FutureDate implements Rule
+{
+    use IsDateRule;
+
+    public function __construct(string $format = 'Y-m-d')
+    {
+        $this->format = $format;
+    }
+
+    public function passes($attribute, $value): bool
+    {
+        $date = $this->createDate($value);
```

Comment:
`createDate` throws an exception is the `$value` is not a valid date. I'd rather have the `passes` function return `false` in this case.